app/views/roasts/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<%= render "shared/partials/nav-global" %>
+<%= render "shared/partials/header-global" %>
+
+<!-- first loop around main and second loop around roasts_brands section -->
+<!-- can loop the markup through Rails since each roast is templated -->
+
+<section class="roasts__slider-wrapper">
+    <label for="bean-slider" aria-label = "Slide to select a roast!">
+      <div class="slider-axis" aria-hidden = "true">
+        <%= image_tag("light-slider-bean.svg", alt:"") %>
+        <span>Light</span>
+        <span></span>
+        <span></span>
+        <span></span>
+        <span></span>
+        <span></span>
+        <span></span>
+        <span>Dark</span>
+        <%= image_tag("dark-slider-bean.svg", alt:"") %>
+      </div>
+    </label>
+    <div class="slider">
+      <input type="range" id = "bean-slider" value="0" min="0" max="8" step="1" aria-valuemin="0" aria-valuemax="8" aria-valuenow="0" class="slider__range">
+    </div>
+</section>
+
+<main class = "roasts">
+  <% roasts.each_with_index do |roast, index| %>
+    <% roastIndex = index %>
+    <section class = "roasts__view" id = "roast<%= index %>">
+      <div class="view__image-wrapper">
+        <%= image_tag(roast.image.thumb('200x200#').url, alt:"") %>
+      </div>
+      <h2><%= roast.name %></h2>
+      <p><strong><%= roast.description %></strong></p>
+      <p><%= roast.sub_description %></p>
+    </section>
+    
+    <% if roast.roast_brands.any? %>
+      <% roast.roast_brands.each_with_index do |brand, index| %>
+        <section class="roasts__brands" id="roast<%= roastIndex %>brand<%= index %>">
+          <h2>Popular Brands</h2>
+          <div class="brand__wrapper">
+            <div class="wrapper__brand">
+              <%=image_tag(brand.image.thumb('200x200#').url, alt:"")%>
+              <h3><%= brand.brand_name %></h3>
+              <p><%= brand.roast_name %></p>
+            </div>
+        </section>
```

Comment:
1. you're missing a closing `</div>` here

2. If you move lines 41-43 out of the brands loop, then you need to also move the closing tags for that HTML outside of the loop (after the `<% end %>` on the next line)